File: spec/code_samples/Php_http1/end_user_authentications@passwords/patch.php
<?php

HttpRequest::methodRegister('PATCH');
$request = new HttpRequest();
$request->setUrl('https://www.directfreight.com/api/end_user_authentications/passwords');
$request->setMethod(HttpRequest::HTTP_METH_PATCH);

$request->setHeaders(array(
  'end-user-token' => 'SOME_STRING_VALUE',
  'content-type' => 'application/json',
  'accept' => 'application/json'
));

$request->setBody('{"new_password":"<ADD STRING VALUE>","old_password":"<ADD STRING VALUE>"}');

try {
  $response = $request->send();

  echo $response->getBody();
} catch (HttpException $ex) {
  echo $ex;
}

File: spec/code_samples/Perl_perl5/end_user_authentications@passwords/patch.pl
#!/usr/bin/perl
use Modern::Perl;
use HTTP::Request::JSON;
use LWP::UserAgent::JSON;
my $request = HTTP::Request::JSON->new(PATCH,
"https://www.directfreight.com/api/end_user_authentications/passwords")
$request->content('{"new_password":"<ADD STRING VALUE>","old_password":"<ADD STRING VALUE>"}');
my $browser = LWP::UserAgent::JSON->new();
my $response = $browser->request($request);
print $response->json_content;